### ISSUE TITLE
Issues understanding the trained model predictions

### ISSUE THREAD

**Issue by andanison** [ncoudray/DeepPATH#68]
Hi

1. I am trying to understand the trained model in `run1a_3D_classifier` which is used for classification between `Normal/LUAD/LUSC` however when I use the model to predict on a new patch, I get logits with 4 outputs which I assume belong to classes `background/Normal/LUAD/LUSC` as label 0 is reserved for the background class. Can you confirm if that is true so that I can interpret the predictions from the model? 

2. Was the checkpoint in `run1a_3D_classifier` trained for patches with 5x or 20x resolution?

**Comment by ncoudray** [comment]
Hi - 

 1. correct
 2. 20x

Best,
Nicolas
